scripts/global_sink_isolation.py

```
@@ -0,0 +1,367 @@
+            violations.append(
+                "%s is called by %s but has no definition under crates/perry-runtime/src — "
+                "the gate cannot classify what it clears." % (helper, RESET_FN)
+            )
+            continue
+        # Follow one level of same-file accessor calls. `test_clear_closure_
+        # side_tables` names no static at all — it goes through
+        # `get_closure_props()` — so a body-only scan classified it as "no
+        # storage" and would not have noticed CLOSURE_PROPS reverting.
+        expanded = helper_body
+        for callee in sorted(set(re.findall(r"\b(get_[a-z0-9_]+)\s*\(", helper_body))):
+            callee_path, callee_body = find_fn_body({path: sources[path]}, callee)
```

Ruff (0.16.1)

[warning] Unpacked variable `callee_path` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)